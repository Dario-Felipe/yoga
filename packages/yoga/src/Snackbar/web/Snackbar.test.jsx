import React from 'react';
import { render } from '@testing-library/react';

import { ThemeProvider, Snackbar } from '../..';

describe('<Snackbar />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <Snackbar />
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
