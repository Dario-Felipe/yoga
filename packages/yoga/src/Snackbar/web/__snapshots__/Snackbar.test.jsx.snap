// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Snackbar /> should match snapshot 1`] = `
.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  margin-right: 8px;
}

.c2 {
  margin: 0;
  padding: 0;
  font-size: 14px;
  line-height: 20px;
  font-weight: 400;
  font-family: Rubik;
  color: #231B22;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-right: 16px;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c3 svg {
  margin-left: 16px;
  margin-right: 0;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  position: absolute;
  bottom: 40px;
  right: 40px;
  min-width: 320px;
  max-width: 600px;
  min-height: 56px;
  max-height: 72px;
  padding: 16px;
  border-radius: 8px;
  box-shadow: 0 2px 6px rgba(152,152,166,0.25);
  background-color: #E0DFFF;
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <svg
        fill="#231B22"
        height="24"
        width="24"
      />
    </div>
    <p
      class="c2"
    />
    <aside
      class="c3"
    />
  </div>
</div>
`;
